### PR TITLE
{data}[foss/2017a] HDF5 v1.10.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2466.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/eae102504bcfa7b4d9c1fdbd9bd25022 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2152.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/db8858c94705b932e0c6a0332fff1ccb for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
ca003.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a56796be9e704a20ea1eeceab9a9ac28 for a full test report.

File: easybuild/easyconfigs/h/HDF5/HDF5-1.10.1-foss-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name = 'HDF5'
+version = '1.10.1'
+
+homepage = 'https://support.hdfgroup.org/HDF5/'
+description = """HDF5 is a unique technology suite that makes possible the management of
+ extremely large and complex data collections."""
```
Comment (verdurin) [inline easybuild/easyconfigs/h/HDF5/HDF5-1.10.1-foss-2017a.eb]: @mmterpstra complained about the language here, but it doesn't bother me.

**Comment by verdurin** [comment]
Merging, thanks @boegel

**Pull request merged**